src/main/java/uk/gov/hmcts/dts/fact/controllers/CourtsController.java

```diff
@@ -55,4 +59,11 @@ public CourtsController(final CourtService courtService) {
     public ResponseEntity<Court> findCourtByName(@PathVariable String slug) {
         return ok(courtService.getCourtBySlug(slug));
     }
+
+    @GetMapping(path = "/search")
+    @ApiOperation("Return court data for download")
+    public ResponseEntity<List<CourtReference>> getCourtsBySearch(@RequestParam("prefix") @Size(min = 1, max = 1) @NotBlank String prefix,
+                                                                  @RequestParam boolean active) {
```

Comment:
I know we discussed about the active flag in the spike meeting....we added that originally because we thought we were going to reuse the admin endpoint. Now this endpoint is separate from the existing admin endpoint and it is only going to be used by the front end....we will never want to return the inactive courts....I wonder we should remove this parameter and always return the active courts?